.gitignore

```diff
@@ -2,6 +2,7 @@
 .facsimile/
 .molecule/
 .vault_pass
+filter_plugins/__pycache__/*
```

Comment:
@srw you should be able to just change that to  `__pycache__` to ignore it everywhere.